brave/brave-core
# Fix toolbar icons.

Relocated toolbar icons to omnibox since toolbar is no longer used in Chromium. Also, relocated settings.icon to chrome/app.

Fixes brave/brave-browser#2641

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
1. Verify Brave icon:
- Start Brave browser and navigate to brave://version,
- Validate that the Brave icon is shown in the location bar to the left of the word "Brave".

2. Verify bookmark icons:
- Start Brave browser and navigate to any site,
- Validate that the bookmark button to the left of the location bar is a ribbon (not a star),
- Click on the bookmark button and verify that it shows as a filled in ribbon (not a filled in star).

3. Verify find in page icon:
- Start Brave browser and navigate to any site,
- Press Ctrl + F to start find in page,
- Validate that the find icon that shows in the location bar looks like a square with a magnifying glass, not a filled in page with magnifying glass.

4. Verify http icon:
- Start Brave browser and navigate to any http site,
- Validate that the icon to the left of "Not secure" is an exclamation point in a filled white circle (not letter i in an unfilled circle).

5. Verify https icons:
- Start Brave browser and navigate to any valid https site,
- Validate that the icon to the left of the address is an unfilled lock with a dot in the middle (not a filled lock).
- Navigate to a bad https site (e.g. https://untrusted-root.badssl.com/),
- Validate that the icon to the left of "Not secure" is a white circle with an x inside (not a red triangle with an exclamation point).

6. Verify settings icon:
- Start Brave browser,
- Click on the Avatar icon and validate that the Settings menu item icon is an unfilled gear (not a filled in gear).

## Reviewer Checklist:
